src/groovy/jobs/AbstractLowDimensionalAnalysisJob.groovy

```diff
@@ -0,0 +1,51 @@
+package jobs
+
+import jobs.steps.BuildTableResultStep
+import jobs.steps.MultiRowAsGroupDumpTableResultsStep
+import jobs.steps.Step
+import jobs.steps.WriteFileStep
+import jobs.steps.helpers.ColumnConfigurator
+import jobs.table.Table
+import org.springframework.beans.factory.annotation.Autowired
+
+import static jobs.steps.AbstractDumpStep.getDEFAULT_OUTPUT_FILE_NAME
+
+abstract class AbstractLowDimensionalAnalysisJob extends AbstractAnalysisJob {
```

Comment:
Why `AbstractLowDimensionalAnalysisJob`? It seems this is used by analyses that take high dimensional data, like the scatter plot.